repos/system_upgrade/el7toel8/actors/ziplconverttobls/actor.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+from leapp.actors import Actor
+from leapp.libraries.common import mounting
+from leapp.libraries.common.config import architecture
+from leapp.libraries.stdlib import CalledProcessError
+from leapp.models import TargetUserSpaceInfo
+from leapp.tags import IPUWorkflowTag, PreparationPhaseTag
+
+
+class ZiplConvertToBLSCFG(Actor):
+    """
+    Convert the zipl boot loader configuration to the the boot loader specification on s390x systems.
+    """
+
+    name = 'zipl_convert_to_blscfg'
```

Comment:
Could you please change the name of the folder to match the actor name, i.e. ziplconverttoblscfg?

Reply:
yep